File: JAVA/Hardware.java
public class Hardware extends Product {
    String brand;
    String model;

    Hardware(){

    }

    Hardware(String brand, String model){
        this.brand = brand;
        this.model = model;
    }

    void setBrand(String brand){
        this.brand = brand;
    }

    void SetModel(String model){
    
        this.model = model;
    }

    String getBrand(){
        return this.brand;
    }

    String getModel(){
        return this.model;
    }

}

File: C++/Hardware.cpp
#include<iostream>
#include<string>
#include "Product.cpp"

using namespace std;

class Hardware : public Product {

    private:
    string brand;
    string model;
    
    

    public :
    Hardware(){

    }


    Hardware(string br, string md){
        brand = br;
        model = md;
        
    }

    
    void setBrand(string br){
        brand = br;
    }

    void setModel(string md){
    
        model = md;
    }

    string getBrand(){
        return brand;
    }

    string getModel(){
        return model;
    }

    ~Hardware(){
        
    }
};
